javascript/mergehistory.js

```diff
@@ -0,0 +1,98 @@
+/*
+    mergehistory.js
+
+    MediaWiki API Demos
+    Demo of `mergehistory` module: Merge the page revisions of Oldpage
+    dating up to 2015-12-31T04:37:41Z into Newpage
+
+*/
+var request = require("request").defaults({jar: true}),
+url = "https://test.wikipedia.org/w/api.php";
+login_token = getLoginToken(); // Step 1
```

Comment:
You've called `getLoginToken()` here and in line 92 as well. I think only one of them is necessary? You may use the one in line 92